app/models/page.rb

```diff
@@ -9,11 +7,12 @@ class Page < ApplicationRecord
   validates :template, inclusion: { in: TEMPLATE_OPTIONS }
   validate :body_present
   validate :unique_slug_including_users_and_orgs, if: :slug_changed?
-  validate :single_landing_page, if: :will_save_change_to_landing_page?
 
   before_validation :set_default_template
   before_save :evaluate_markdown
-  after_save :bust_cache
+
+  after_commit :ensure_uniqueness_of_landinge_page
+  after_commit :bust_cache
```

Comment:
This is not strictly related to this PR but we should get in the habit of enqueing async workers only when Rails has committed its transaction, see also https://github.com/forem/forem/pull/13969